### ISSUE TITLE
rm parameter -rf unknow error

### ISSUE THREAD

**Issue by TaoWolf** [microsoft/terminal#3870]
![image](https://user-images.githubusercontent.com/16403888/70333943-6f926600-187f-11ea-962a-7f5fd6b78cdd.png)

**Comment by zadjii-msft** [comment]
I don't think that `-r` or `-f` are arguments that are accepted by the powershell command `Remove-Item`, which is aliased to `rm` in powershell. You're probably thinking of the linux command `rm` which does the same thing, but has different parameters. 

I'd try `rm -recurse -force kkk`.
